packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -31,14 +31,17 @@ export class FlowBuilderApi {
     const newApi = new FlowBuilderApi()
 
     newApi.url = options.url
-    newApi.flowUrl = options.flowUrl
-    newApi.flow = options.flow ?? (await newApi.getFlow(options.accessToken))
     newApi.request = options.request
 
     if (process.env.NODE_ENV === ProcessEnvNodeEnvs.DEVELOPMENT) {
       await newApi.updateSessionWithUserInfo(options.accessToken)
     }
-
+    const updatedRequest = newApi.request
+    newApi.flowUrl = options.flowUrl.replace(
+      '{bot_id}',
+      updatedRequest.session.bot.id
+    )
+    newApi.flow = options.flow ?? (await newApi.getFlow(options.accessToken))
```

Comment:
to make it work correctly in local, we should retrieve botId from userInfo, not from incoming session.bot.user.id, because it will be the default for webchat dev ('00000')
This should be refactored at some point

Reply:
added a TODO for refactoring